docker-compose.yaml

```diff
@@ -1,14 +1,19 @@
 version: "3"
 services:
+
   mumble:
-    build: mumble/.
+    image: docker.pkg.github.com/flipdot/mumble-docker/mumble:latest
+    command: /usr/bin/murmurd -v -fg -ini /etc/murmur/murmur.ini
```

Comment:
why defined it here, when we have CMD in the Dockerfile?

Reply:
added it for the "supw" flag which isn't needed...

But i'll replace all the murmurs with mumble, so we'll need it again :)

Reply:
ok done